### PR TITLE
Feature#13 loging admin

### PR DESCRIPTION
**Pull request by oscaregr**
# Description

What did you implemented/Why did you implemented this:

- What?: I've added login interface for admin, the respective controller than permit go to admin index when the pasword is correct .
- Why?: this is necesary for not permit changes for anybody without authorization

## Screenshots
image

### CODE REVIEW

**Review by DiegoMoralesBravo — APPROVED** [review]

**Review by DiegoCoronaBarragan — APPROVED** [review]
LGTM

**Review by Yaxkin121 — APPROVED** [review]
lgtm

**Review by MarcoRodriguez23 — APPROVED** [review]
LGFM !!!

**Review by Reiyirak — APPROVED** [review]
LGTM

**Review by AangelF — APPROVED** [review]

File: app/assets/stylesheets/base/_base.scss [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
-  	font-size: 18px;
+  background-color: $--primary-color-dark;
+  font-size: 18px;
 }
+
+a {
+  text-decoration:none;
+}
+
+.container{
```
Comment (JuanequeX) [inline app/assets/stylesheets/base/_base.scss:12]: add a space between the class name and your curly brace

**Review by CarlosSe — APPROVED** [review]
LGTM 🚀

File: app/assets/stylesheets/views/_checkView.scss [hunk 2]
```diff
@@ -1,71 +1,80 @@
+@import 'utilities/variables';
```
Comment (JuanequeX) [inline app/assets/stylesheets/views/_checkView.scss:1]: this import here is unnecessary

File: app/assets/stylesheets/views/_loginView.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import 'utilities/variables';
```
Comment (JuanequeX) [inline app/assets/stylesheets/views/_loginView.scss:1]: this is unnecessary

File: app/helpers/admin_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module AdminHelper
+end
```
Comment (JuanequeX) [inline app/helpers/admin_helper.rb:2]: if this helper are empty please remove it

File: app/helpers/login_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module LoginHelper
+end
```
Comment (JuanequeX) [inline app/helpers/login_helper.rb:2]: if this helper are empty please remove it

File: app/views/admin/index.html.erb [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (JuanequeX) [inline app/views/admin/index.html.erb:1]: What is the reason to have this file with any content?

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by juperameza — CHANGES_REQUESTED** [review]
Attend @JuanequeX request
And also why are you doing the log in manually? Shouldn't it be better if you use something like this  [gem](https://github.com/heartcombo/devise)

**Comment by JuanequeX** [comment]
Tengo la duda, porque todas dan LGTM sin revisar?

File: app/helpers/login_helper.rb (hunk 5 above)
Comment (oscaregr) [reply app/helpers/login_helper.rb:2]: all the files?

File: app/assets/stylesheets/base/_base.scss (hunk 1 above)
Comment (oscaregr) [reply app/assets/stylesheets/base/_base.scss:12]: done

File: app/assets/stylesheets/views/_loginView.scss (hunk 3 above)
Comment (oscaregr) [reply app/assets/stylesheets/views/_loginView.scss:1]: if we no import the variables, rails no find them, and send this in the browser, anyway that happen if we put the import in the main.scss

image
image

File: app/views/admin/index.html.erb (hunk 6 above)
Comment (oscaregr) [reply app/views/admin/index.html.erb:1]: @JuanequeX for redirect to one file when de admin login successfully, and not made conflict when the teammate with that issue does his merge

File: app/assets/stylesheets/views/_loginView.scss (hunk 3 above)
Comment (JuanequeX) [reply app/assets/stylesheets/views/_loginView.scss:1]: how looks your principal file of scss?

File: app/helpers/admin_helper.rb (hunk 4 above)
Comment (oscaregr) [reply app/helpers/admin_helper.rb:2]: done

File: app/assets/stylesheets/views/_loginView.scss (hunk 3 above)
Comment (oscaregr) [reply app/assets/stylesheets/views/_loginView.scss]: reolved

File: app/views/admin/index.html.erb (hunk 6 above)
Comment (oscaregr) [reply app/views/admin/index.html.erb:1]: deleted file

File: app/assets/stylesheets/views/_checkView.scss (hunk 2 above)
Comment (oscaregr) [reply app/assets/stylesheets/views/_checkView.scss]: done

File: app/assets/images/back.svg [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+<svg fill="#000000" height="35px" width="35px" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" 
+	 viewBox="0 0 477.175 477.175" xml:space="preserve">
+<g>
+	<path d="M145.188,238.575l215.5-215.5c5.3-5.3,5.3-13.8,0-19.1s-13.8-5.3-19.1,0l-225.1,225.1c-5.3,5.3-5.3,13.8,0,19.1l225.1,225
+		c2.6,2.6,6.1,4,9.5,4s6.9-1.3,9.5-4c5.3-5.3,5.3-13.8,0-19.1L145.188,238.575z"/>
+</g>
+</svg>
```
Comment (juperameza) [inline app/assets/images/back.svg:9]: Add a blank line at the end of this file

File: app/assets/stylesheets/layout/_body.scss [hunk 8]
```diff
@@ -0,0 +1,8 @@
+.body {
+  background-color: $--primary-color-dark;
+  font-size: 18px;
+}
+
+.a {
+  text-decoration:none;
+}
```
Comment (juperameza) [inline app/assets/stylesheets/layout/_body.scss:8]: Same here blank like at the end

**Review by juperameza — CHANGES_REQUESTED** [review]
Blank lines

**Review by juperameza — APPROVED** [review]
Lgtm

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**